ava-labs/core-extension
# fix: show balances for custom tokens too

## Description
* We stopped balances for the custom tokens after migrating to VM Modules

## Testing
* Add a custom token
* Verify you see its balance

## Screenshots:
![image](https://github.com/user-attachments/assets/cc7f0cbd-d81c-45af-acf9-9bd5dfa57b44)

## Checklist for the author
